mpl_interactions/xarray_helpers.py

```diff
@@ -0,0 +1,159 @@
+import numpy as np
+
+
+def choose_fmt_str(dtype=None):
```

Comment:
I will move to `helpers.py`. Im going to open a new issue about formatting sliders since there are many ways this function could be expanded and it could/should be called by like all the `iplt` functions.